tests/ShareUrlTest.php

```diff
@@ -36,9 +36,9 @@ public function testShareUrls() {
     $this->assertEquals('https://www.facebook.com/sharer.php', $fb['href']);
     $this->assertStringContainsString('node/' . $node->nid, $fb['query']['u']);
 
-    $this->assertEquals('http://twitter.com/share', $twitter['href']);
-    $this->assertStringContainsString('node/' . $node->nid, $twitter['query']['text']);
-    $this->assertStringContainsString('utm_campaign=%5B' . $node->nid . '%5D', $twitter['query']['text']);
+    $this->assertEquals('https://twitter.com/intent/tweet', $twitter['href']);
+    $this->assertContains('node/' . $node->nid, $twitter['query']['text']);
+    $this->assertContains('utm_campaign=%5B' . $node->nid . '%5D', $twitter['query']['text']);
```

Comment:
I think this should stay `assertStringContainsString()` :thinking:

Reply:
I‘ve reverted that part of the change.